library/test/src/test_result.rs

```diff
@@ -7,11 +11,16 @@ use super::types::TestDesc;
 
 pub use self::TestResult::*;
 
-// Return codes for secondary process.
+// Return code for secondary process.
 // Start somewhere other than 0 so we know the return code means what we think
 // it means.
 pub const TR_OK: i32 = 50;
-pub const TR_FAILED: i32 = 51;
+
+#[cfg(unix)]
+const SIGABRT: i32 = 6;
```

Comment:
Shouldn't we use `libc::SIGABRT`? Even `std` doesn't hard-code this.

Reply:
I can do that, have to add a dep though.